src/Snap/Snaplet/PureScript.hs

```diff
@@ -55,7 +55,8 @@ initPurs = makeSnaplet "purs" description (Just dataDir) $ do
   bundleOpts  <- liftIO (lookupDefault mempty config "bundleOpts")
   modules     <- liftIO (lookupDefault ["Main"] config  "modules")
   psPath      <- liftIO (lookupDefault mempty config "pureScriptPath")
-  spagoPath   <- findOrInstallSpago psPath =<< liftIO (Cfg.lookup config "spagoPath")
+  spagoPath   <- liftIO (Cfg.lookup config "spagoPath") >>=
+    \sPath -> findOrInstallSpago psPath sPath
```

Comment:
Thanks! Is this related to deep subsumption and, if it is, should we add a comment explaining why that has to be eta expanded?

Reply:
Yes, exactly; I'll add a line saying this has to be eta expanded due to deep subsumption and link to the proposal PR.

Reply:
Wonderful, thanks!